### DISCUSSION TITLE
error ketika install rquirements (library)

### DISCUSSION THREAD

**Discussion by AjiPramasto** [teamalgoritma/community#176] [Q&A] [answered]
Halo
mau bertanya ketika install library terjadi error seperti terlampir,
mohon bantuannya.
terima kasih
<img width="676" alt="error install library" src="https://user-images.githubusercontent.com/107389614/177798658-425cc0bd-aae7-4056-840d-3d6b30ac4c45.PNG">

**Comment by triarm** [comment]
Hi @AjiPramasto ,
Untuk versi Python pada environment capstone-flask apakah sudah pada versi 3.8? Jika belum silahkan untuk mengganti versi Python atau membuat environment baru dengan versi Python 3.8. Kemudian jika sudah mengganti versi Pythonnya silahkan untuk melakukan instalasi package kembali.

Demikian yang dapat kami sampaikan. Semoga membantu.

**Reply by AjiPramasto** [reply]
untuk versi Python sudah sesuai mas
![image](https://user-images.githubusercontent.com/107389614/177926094-ebf205db-d67b-4e53-9082-9656d5cd215b.png)

**Comment by amaliasf** [comment]
Hi Pak @AjiPramasto ,
Boleh dicoba terlebih dahulu untuk melakukan `pip install pandas==1.3.4`. 
Kalau masih error bisa dicoba kembali install tanpa men-spesifikasikan version, yaitu dengan `pip install pandas`

Kalau masih error juga, silahkan lampirkan pesan error nya Pak. Terimakasih

**Reply by AjiPramasto** [reply]
masih error juga mas, btw untuk kebutuhan capstone dashboard perlu library tidak hanya pandas (reminder aja) dan itu error semua pas install,
berikut ya
![image](https://user-images.githubusercontent.com/107389614/177949589-14997eba-e288-4059-aa59-7158c4f542a4.png)
![image](https://user-images.githubusercontent.com/107389614/177949665-3cbe17ed-f388-4a67-b049-a60a24fad1f8.png)

**Reply by AjiPramasto** [reply]
Sorry mba...kirain masih mas Tria

**Reply by amaliasf** [reply]
Halo Pak @AjiPramasto 
Dikarenakan Bapak menggunakan laptop kantor, dugaan kami ada pembatasan koneksi di laptopnya. Mungkin bisa disampaikan pada teknisi nya untuk membuka akses nya terlebih dahulu. Semoga bisa membantu Pak :)

**Reply by AjiPramasto** [reply]
Bisa lebih spesifik kah seperti apa untuk perbaikan koneksinya atau yang lainnya mungkin, karena saya coba di rumah (tidak pakai vpn dsb) tetap tidak bisa

**Reply by AjiPramasto** [reply]
tambahan juga,
ini saya ikut program fullstack, jadi masih ada 2 kategori training ke depan, mohon diinfo sebelumnya untuk penginstallan program/aplikasi sehingga jika ada kendala teknis seperti ini bisa langsung ketahuan dan cepat tertangani

terima kasih

**Comment by yosiaazarya** [comment, marked as answer]
Dear pak Aji, mohon dipastikan ke tim IT di kantor bapak, apakah laptop bapak mengalami pembatasan dari segi instalasi ataupun mengalami pembatasan untuk melakukan instalasi dari internet? Karena hal ini kerap terjadi pada laptop kantor Bapak. Hal ini dapat kami sampaikan karena melihat error message yang tertampil yaitu berkaitan dengan gagalnya untuk connect ke server untuk download librarynya. Terima kasih pak.
